src/api/app/models/project_status/pack_info.rb

```diff
@@ -74,7 +74,7 @@ def relationships_to_xml(builder, arrayname, elementname, tag)
     end
 
     def set_versrel(versrel, time)
-      return if @versiontime && @versiontime > time
+      return if @versiontime && @versiontime.to_i > time.to_i
```

Comment:
Nitpick, but reading the commit description I would say this belongs to the previous commit that introduced the delegate:-)